crt_portal/cts_forms/templates/forms/report_base.html

```diff
@@ -14,13 +14,15 @@
     <div class="grid-row grid-gap">
       <div class="tablet:grid-col-8 tablet:grid-offset-2">
         {% block page_info_card %}{% endblock %}
+        {% block form_id %}
         <form id="report-form"
               class="usa-form"
               action="/report/"
```

Comment:
Not changed in this PR but noting here that this is a good spot to use [{% url %}](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#url) as opposed to hard coding the path.